arraycontext/impl/pyopencl/fake_numpy.py

```diff
@@ -58,6 +60,17 @@ def _get_fake_numpy_linalg_namespace(self):
     # These operations provide access to numpy-style comparisons in that
     # case.
 
+    def __getattr__(self, name):
+        print(name)
+        cl_funcs = ["abs", "sin", "cos", "tan", "arcsin", "arccos", "arctan",
+                    "sinh", "cosh", "tanh", "exp", "log", "log10", "isnan",
+                    "sqrt", "exp"]
+        if name in cl_funcs:
+            from functools import partial
+            return partial(rec_map_array_container, getattr(cl, name))
+
+        return super().__getattr__(name)
```

Comment:
This should be necessary. For the PyOpenCLArrayContext, I don't see a big problem in calling loopy kernel for these operations.

Reply:
Makes sense let me get rid of it then